tests/network-ovn

```diff
@@ -19,16 +19,21 @@ waitInstanceReady() (
   set +x
   # shellcheck disable=SC3043
   local instName="$1"
+  # shellcheck disable=SC3043
+  local instProj="${2:-}"
+  if [ -z "${instProj}" ]; then
```

Comment:
What does this bit do?

Reply:
Does it look for current project? If so why don't we just drop the --project flag if no project arg ?

Reply:
This allows to report the project's name in case of timeout. I could update the error message to mention the instance in the currently selected project could get ready?

Reply:
ah ok, no thats fine, but a comment explaining why its there will suffice, thanks!